.env.local.example

```
@@ -0,0 +1,34 @@
+# then use its real Client Key (<key>@<host>:8080/<project_id>).
+SENTRY_DSN=http://e4600773d54141d8b8e8250fc480f168@localhost:8080/1
+LOG_LEVEL=debug
+
+# Variables required by `docker compose` (see the "Local stack" section in
+# AGENTS.md). When shelling out to Compose, export these (or place them in the
+# project `.env`) so the stack starts without committing credentials.
+POSTGRES_USER=anidev
+POSTGRES_PASSWORD=anidev
+POSTGRES_DB=anidev
+RUSTRAK_SESSION_SECRET_KEY=667e0e851a0df3f47d1929a54638dc9bad8563606789bd10c4a7c091070db092
+RUSTRAK_SUPERUSER=admin@example.com:admin
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)